ODC new features and known issues

Preeticp opened the pull request:
This PR adds the new features and known issues for ODC 4.6
Please note: 

- Some of the new features are commented out as they are dependent on the async release of Pipelines on 28th. I will create a fresh PR to comment them in post the release. 
- In the Known Issue, one issue is commented out as the QE is verifying it and I will get confirmation on this only by tomorrow. 
This is the only one pending QE ack, rest all have been verified by the devs and the QE.

karthikjeeyar commented on release_notes/ocp-4-6-release-notes.adoc:
```diff
@@ -433,6 +433,30 @@ from both sides of the modal are available for display at one time. Clicking
+[id="ocp-4-6-web-console-administrator-perspective"]
+==== Administrator Perspective
+
+* The *Pipelines* view was updated to provide details on triggers, event listeners, trigger templates, trigger bindings, and cluster trigger bindings.
```
The *Pipelines* navigation item was updated to provide tab views for Pipelines, Tasks and Triggers. Triggers will include tabs for Event Listeners, Trigger Templates, Trigger Bindings, and Cluster Trigger Bindings.

invincibleJai commented on release_notes/ocp-4-6-release-notes.adoc:
```diff
@@ -433,6 +433,30 @@ from both sides of the modal are available for display at one time. Clicking
+[id="ocp-4-6-web-console-developer-perspective"]
+==== Developer Perspective
+
+* Based on user credentials the user is now directed to either the *Administrator* or *Developer*  perspective and an interactive tour of the functionalities for the perspective  is provided.
```
should this be 
```suggestion
* Based on user access roles/privileges the user is now directed to either the *Administrator* or *Developer*  perspective and an interactive tour of the functionalities for the perspective  is provided on Developer Perspective.
```

invincibleJai commented on release_notes/ocp-4-6-release-notes.adoc:
```diff
@@ -433,6 +433,30 @@ from both sides of the modal are available for display at one time. Clicking
+* The *Developer Catalog* now displays only those Helm charts that are compatible with the OpenShift version and filters out charts that are not compatible with the user's cluster.
+* The Knative eventing workflow was enhanced:
+** Support for Knative Eventing Channels to build a reliable event delivery mechanism has been added.
+** You can now create connectors and also specify the sink as a Knative Service or a URI when creating Event Sources.
```
```suggestion
** You can now create Subscription for channels and Triggers with associated filters for Brokers and Select knative service as subscriber
** You can now specify the sink as any Knative resources i.e knative service, chanel or broker from that namespace or a URI when creating Event Sources.
** You can now visualise the relationship for eventSource subscribed by knative service through channel/subscription or broker/trigger, details as well can be seen in sidepanel 
```

invincibleJai commented on release_notes/ocp-4-6-release-notes.adoc:
```diff
@@ -1263,6 +1291,16 @@ Once the firewall rule of the machine with the missing infrastructure ID is remo
+
+* Pipeline runs with skipped tasks incorrectly show the tasks as *Failed*. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1880389[1880389])
+
+// confirmation on this one needed* The Kiali link can return false positive responses as the Service Mesh operator <replace with Elastic Search operator?> is unavailable in the new bundle format yet. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1866035[1866035])
+
```
this would be closed IMO cc @gajanan-more

Preeticp replied:
Ack, removing this.

Preeticp commented on release_notes/ocp-4-6-release-notes.adoc:
```diff
@@ -433,6 +433,30 @@ from both sides of the modal are available for display at one time. Clicking
+* The *List* view and *Topology* view now provide the same information so that the user can choose the best view based on the application size and number of components.
+* Support for all workload types is now provided in the *Topology* and the *List* view to get a better idea of the compute resources being used.
+* You can now select the Helm chart version and the application version while installing the chart from the *Developer catalog*. You can also switch between the form and YAML editor while preserving the values that have been entered .
+* The *Developer Catalog* now displays only those Helm charts that are compatible with the OpenShift version and filters out charts that are not compatible with the user's cluster.
```
Cool, will remove this point then. Thank you.

codyhoag reviewed (commented):
Just a few suggestions

Preeticp commented on release_notes/ocp-4-6-release-notes.adoc:
```diff
@@ -583,6 +583,33 @@ from both sides of the modal are available for display at one time. Clicking
+* An interactive getting started tour of the functionalities in the *Developer* perspective is now provided when a user logs in.
+* The *List* view and *Topology* view now provide the same information so that the user can choose the best view based on the application size and number of components.
+* Support for all workload types is now provided in the *Topology* and the *List* view to get a better idea of the compute resources being used.
+* The *Developer Catalog* now displays only those Helm charts that are compatible with the OpenShift version and filters out charts that are not compatible with the user's cluster.
```
I thought I had removed this point. Removing it as per SME input: https://github.com/openshift/openshift-docs/pull/26488#discussion_r505337520

Preeticp commented:
@codyhoag I have fixed all the review comments. We received validation from QE for the Helm 3rd party charts KI and I have removed it. I have also squashed the commits. Please merge if you think it's good. Thank you!

codyhoag commented:
Thanks, @Preeticp! Looks good! I just did a force push to fix some merge conflicts (RNs are changing constantly). I will merge now.

Preeticp commented:
Thank you so much @codyhoag
